packages/common/core-interfaces/package.json

```diff
@@ -24,19 +55,20 @@
 		}
 	},
 	"main": "dist/index.js",
-	"module": "lib/index.mjs",
 	"types": "dist/index.d.ts",
 	"scripts": {
 		"api": "fluid-build . --task api",
-		"api-extractor:commonjs": "api-extractor run --local",
-		"api-extractor:esnext": "api-extractor run --config ./api-extractor-esm.json",
+		"api-extractor:commonjs": "api-extractor run --config ./api-extractor-cjs.json",
+		"api-extractor:esnext": "api-extractor run --local",
 		"build": "fluid-build . --task build",
 		"build:commonjs": "fluid-build . --task commonjs",
 		"build:compile": "fluid-build . --task compile",
 		"build:docs": "fluid-build . --task api",
-		"build:esnext": "tsc-multi --config ../../../common/build/build-common/tsc-multi.esm.json",
-		"build:test": "tsc --project ./src/test/tsconfig.json",
-		"check:are-the-types-wrong": "attw --pack",
+		"build:esnext": "tsc --project ./tsconfig.json",
+		"build:test": "npm run build:test:esm && npm run build:test:cjs",
+		"build:test:cjs": "fluid-tsc commonjs --project ./src/test/tsconfig.cjs.json",
+		"build:test:esm": "tsc --project ./src/test/tsconfig.json",
+		"check:are-the-types-wrong": "attw --pack . --entrypoints .",
```

Comment:
any idea on why attw script wasn't this before?

Reply:
I do know.  It is because attw's matrix of tests includes reachability from node10, which only works for the '.' export (via main & types).  When we expand the exports to include things like './alpha' and './beta' attw fails because these are not reachable from node10 [unless we start doing unnatural things.]

Unfortunately, attw doesn't have an option to ignore node10 for these extra exports, so instead we have to have to constraint it to checking just the '.' entrypoint.

Part of my automatic conversion is to run 'attw -P' and verify that every (except node10) works as expected.